docs/connect-blockchain/cosmos.mdx

```diff
@@ -0,0 +1,377 @@
+---
+title: Integrate Web3Auth with the Cosmos Blockchain
+sidebar_label: Cosmos
+image: "content-hub/guides/banners/cosmos.png"
```

Comment:
Please add this image.

Reply:
Renamed the image to the correct file name,